src/utils/clmfates_interfaceMod.F90

```diff
@@ -197,6 +197,11 @@ module CLMFatesInterfaceMod
    ! developer will at least question its usage (RGK)
    private :: hlm_bounds_to_fates_bounds
 
+   ! The GetAndSetFatesTimeis function is used to get the current time from the CIME 
```

Comment:
Small request to change use of CIME to CLM. The functions being used are the CLM specific ones, not the CIME ones.

Reply:
Thanks for the correction and clarification @ekluzek.